src/macrofier/engine.mjs

```diff
@@ -1080,6 +1084,7 @@ function insertMethodMacros(template, methodObj, json, templates, examples={}) {
     .replace(/\$\{method\.params\.count}/g, methodObj.params ? methodObj.params.length : 0)
     .replace(/\$\{if\.params\}(.*?)\$\{end\.if\.params\}/gms, method.params.length ? '$1' : '')
     .replace(/\$\{if\.context\}(.*?)\$\{end\.if\.context\}/gms, event.params.length ? '$1' : '')
+    .replace(/\$\{method\.params\.instantiation\}/g,  paramInst)
```

Comment:
I think maybe `instantiation` isn't the best name for parameters (it works great for result, though)... we're actually instantiating the JSON versions of the params here, right?

Maybe this macro can be called something like one of these:

- `${method.params.json-values}`
- `${method.params.serialization}`
- `${method.params.transport}`

Thoughts?